README.md

```diff
@@ -1,70 +1,82 @@
 
-<br>
 
-<div align="center">
-  <img src="https://forthebadge.com/images/badges/for-you.svg" />
-  <img src="https://forthebadge.com/images/badges/made-with-markdown.svg" alt="css">
-  <img src="https://forthebadge.com/images/badges/made-with-javascript.svg" alt="js">
-  <img src="https://forthebadge.com/images/badges/open-source.svg" alt="os">
-  <img src="https://forthebadge.com/images/badges/check-it-out.svg" alt="check">
-  <img src="" alt="">
-  <img src="https://forthebadge.com/images/badges/built-by-developers.svg" />
-</div>
+![image](https://user-images.githubusercontent.com/54996800/192564253-f3373ecc-bc86-4c13-a0bd-551b9c23f788.png)
 
-## Setup Steps
+[![GitHub issues](https://img.shields.io/github/issues/vedant-jain03/certificate-generator?style=for-the-badge&color=important)](https://github.com/vedant-jain03/certificate-generator/issues)
+[![GitHub forks](https://img.shields.io/github/forks/vedant-jain03/certificate-generator?style=for-the-badge&color=blue)](https://github.com/vedant-jain03/certificate-generator/fork)
+[![GitHub stars](https://img.shields.io/github/stars/vedant-jain03/certificate-generator?style=for-the-badge&color=yellow)](https://github.com/vedant-jain03/certificate-generator/stargazers)
+</div>
+</br>  
 
-- Go to directory
-```
-$ cd certificate-generator
-```
-- Install Dependencies
-```
-$ npm install
-```
-- Rename .env.example to .env and add your Firebase api key
-```
-$ mv .env.example .env
-```
-- Start LocalHost Server
-```
-$ npm run start
-```
+<h2> What is Hacktoberfest? </h2>
+<p>HACKTOBERFEST IS DIGITALOCEAN’S ANNUAL EVENT THAT ENCOURAGES PEOPLE TO CONTRIBUTE TO OPEN SOURCE THROUGHOUT OCTOBER. MUCH OF MODERN TECH INFRASTRUCTURE—INCLUDING SOME OF DIGITALOCEAN’S OWN PRODUCTS—RELIES ON OPEN-SOURCE PROJECTS BUILT AND MAINTAINED BY PASSIONATE PEOPLE WHO OFTEN DON’T HAVE THE STAFF OR BUDGETS TO DO MUCH MORE THAN KEEP THE PROJECT ALIVE. HACKTOBERFEST IS ALL ABOUT GIVING BACK TO THOSE PROJECTS, SHARPENING SKILLS, AND CELEBRATING ALL THINGS OPEN SOURCE, ESPECIALLY THE PEOPLE THAT MAKE OPEN SOURCE SO SPECIAL.
 
-## Tech Stack Used
-  - Javascript
-  - React
-  - Firebase
-  - Material UI
+FOR THE PAST 9 YEARS, THOUSANDS OF PEOPLE—CODERS AND NON-CODERS ALIKE—HAVE PARTICIPATED IN HACKTOBERFEST TO SUPPORT THE PROJECTS THEY USE AND LOVE, LEARN AND PRACTICE SKILLS THAT WILL ENHANCE THEIR CAREERS, AND MEET NEW PEOPLE WHO LOVE OPEN SOURCE AS MUCH AS THEY DO.</p>
+</br> 
 
-# [Code of Conduct](CODE_OF_CONDUCT.md)
+## Hacktoberfest 2022 : Why should you participate?
+<a href="https://www.youtube.com/watch?v=LruUhWylI5w"><img src="https://user-images.githubusercontent.com/54996800/193015811-02f5678b-ca69-4878-936f-85aaf1042daf.png" width="100"/></a>
+</br> 
 
+## Values
+- Everyone is welcome 
+- It's enjoyable and you can improve your skills
+- You'll be meeting a lot of new people along the way.
+</br> 
 
-# [Getting Started](CONTRIBUTING.md)
+## What are the rules for participating?
```

Comment:
This is already and well written in hacktoberfest website, improving the above changes will reference to this, you can remove this!
WDYT?